src/event.rs

```diff
@@ -97,210 +94,218 @@ pub enum Key {
     Esc,
 
     #[doc(hidden)]
-    __IsNotComplete
+    __IsNotComplete,
 }
 
 /// Parse an Event from `item` and possibly subsequent bytes through `iter`.
-pub fn parse_event<I>(item: Result<u8, Error>, iter: &mut I) -> Result<Event, Error>
```

Comment:
This method was *always* called with Ok(_) as first parameter, so it's a bit cleaner to directly take a `u8` instead.